### ISSUE TITLE
Missing chapters and incorrect formatting on certain verses in al English translations

### ISSUE THREAD

**Issue by AlphaHasher** [martijnlentink/propresenter-custom-bibles#4]
Hello, I'm opening another issue as this is a non related issue to the other one I opened earlier today

As the title says I noticed that some of the verses were straight up broken with several chapters missing

![Screenshot 2024-02-18 at 6.42.38 PM.png](https://github.com/martijnlentink/propresenter-custom-bibles/assets/33443001/355f2d2e-0e5c-4b88-8e08-85ca9a2c79d7)

![Screenshot 2024-02-18 at 6.44.19 PM.png](https://github.com/martijnlentink/propresenter-custom-bibles/assets/33443001/a9ff9165-73c9-4c10-96d5-b696716131a2)

![Screenshot 2024-02-18 at 6.45.12 PM.png](https://github.com/martijnlentink/propresenter-custom-bibles/assets/33443001/aeb9c641-82e1-4b66-99d0-c60a2ba98f6a)

**Comment by martijnlentink** [comment]
Thanks for taking the time to provide some screenshots, this really helps!

I was able to reproduce and was able to figure out what was happening. Long story short is the the formatting that bibles can perform in the USX format is varying with quite some options. I needed to dig a bit into the documentation to find a way how to catch all the cases. Now I think I have rewritten it such that it will be able to handle quite some more formatting options, but I still have to test if older bibles will work as before.

I committed the changes but will take some time this week to verify it and make binaries if so works as expected.
